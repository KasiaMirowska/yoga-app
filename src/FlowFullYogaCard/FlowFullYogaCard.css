.bt-container {
    margin: 20px;
}
.pose-info {
    background-color: rgb(233, 224, 231); 
    text-align: center; 
    padding-top: 30px; 
    padding-bottom: 30px;
    margin: auto;
    margin-bottom: 40px;
    filter: drop-shadow(0 0 0.75rem rgb(151, 132, 136));
}
.iframe {
    width: 80%;
}
.text-container2{
    width: 75%
}
@media (min-width: 800px) {
    .pose-info {
        max-width: 550%;
        background-color: rgb(233, 224, 231); 
        padding-top: 30px; 
        padding-bottom: 30px; 
        margin: auto;
        margin-bottom: 40px;
        filter: drop-shadow(0 0 0.75rem rgb(151, 132, 136));
    }
    
  }  