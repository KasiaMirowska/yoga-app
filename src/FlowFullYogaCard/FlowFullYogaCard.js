import React from 'react';
import YogaContext from '../Context';



export default class FlowFullYogaCard extends React.Component {
    static contextType = YogaContext;
    
    render(){
        console.log('HERE', this.context)
        return(
            <div>
                hi flow
                this.context.currentFlow.name
            </div>
        )
    }
}