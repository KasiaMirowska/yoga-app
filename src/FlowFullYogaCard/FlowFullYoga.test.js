import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import FlowFullYogaCard from './FlowFullYogaCard';
import {BrowserRouter as BR} from 'react-router-dom';

describe('FlowFullYogaCard component', () => {
    let props = {
        location: { pathname: '/flow'},
        history: {location: {pathname: '/flow'}},
        match: {params: {pose_id: 1}}
    };
   
    it ('renders without crashing',() => {
        const div = document.createElement('div');
        ReactDom.render(<BR><FlowFullYogaCard {...props} /></BR>, div);
        ReactDom.unmountComponentAtNode(div);
    });
   
    it('renders UI as expected', () => {
        const fullCard = renderer.create(<BR><FlowFullYogaCard {...props} /></BR>);
        expect(fullCard.toJSON()).toMatchSnapshot();
    });
});