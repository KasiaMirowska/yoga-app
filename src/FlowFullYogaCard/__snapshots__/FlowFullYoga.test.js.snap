// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlowFullYogaCard component renders UI as expected 1`] = `
<div
  className="pose-info"
>
  <h3
    className="title"
  />
  <h3
    className="title"
  />
  <div
    className="text-container"
  >
    <p>
      BENEFITS : 
    </p>
    <br />
    <p>
      LEVEL : 
    </p>
    <br />
    <p>
      POSE TYPE : 
    </p>
    <br />
  </div>
  <iframe
    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
    allowFullScreen={true}
    frameBorder="0"
    height="315"
    title="yoga pose instructions"
    width="560"
  />
  <div
    className="bt-container"
  >
    <button
      onClick={[Function]}
    >
      Back
    </button>
  </div>
</div>
`;
